Makefile

```diff
@@ -46,4 +46,4 @@ endif
 	go build -o bin/tidb-dashboard -ldflags '$(LDFLAGS)' -tags "${BUILD_TAGS}" cmd/tidb-dashboard/main.go
 
 run:
-	bin/tidb-dashboard --debug --experimental
+	bin/tidb-dashboard --debug --experimental --non-root-login
```

Comment:
How about supporting using env var to enable/disable such features.

Reply:
cmd options is convenient as the switch. But like https://github.com/pingcap/tidb-dashboard/pull/985#discussion_r702553685 says here `non-root-login` is not required to be provided by the user, we're using it as a feature switch, and it's `tidb-dashboard` *library* switch.

Reply:
Yep, it could be better if we can support that.